tests/unit/auth/v2/queryAuthCheck.js

```diff
@@ -11,18 +12,111 @@ describe('v2: queryAuthCheck', () => {
         { token: undefined, error: false },
         { token: 'invalid-token', error: true },
         { token: 'a'.repeat(128), error: false },
-    ].forEach(test => it(`test with token(${test.token})`, () => {
-        const request = { method: 'GET' };
+    ].forEach(test => {
+        it(`test with token(${test.token})`, () => {
+            const request = { method: 'GET' };
+            const data = {
+                SecurityToken: test.token,
+            };
+            const res = queryAuthCheck(request, log, data);
+            if (test.error) {
```

Comment:
I would advise against conditionals in tests, instead create a test for each condition.

Reply:
I agree, although that's not supposed to be a part of my changes - only seems as new addition cause I separated line 16 from line 15 for better readability per style guidelines. 
It seems like the same logic replicated along other places such as: https://github.com/scality/Arsenal/blob/development/7.7/tests/unit/auth/v4/headerAuthCheck.js#L47 ,  https://github.com/scality/Arsenal/blob/development/7.7/tests/unit/auth/v2/headerAuthCheck.js#L21